### PR TITLE
add certificate section to doc

### PR DESCRIPTION
**Pull request by dacolgit**

### CODE REVIEW

File: docs/modules/publisher-migrationpath.md [hunk 1]
```diff
@@ -128,7 +128,22 @@ The following table describes the command line arguments, which were available i
...
+
+However, the biggest hurdle most OT admins need to overcome when deploying the Azure Industrial IoT platform is to accept the Azure Industrial IoT platform X.509 certificate in the OPC UA server the platform tries to connect to. There is usually a configuration tool that comes with the built-in OPC UA server where certificates can be trusted.
+For example for KepServerEx, configure the trusted Client certificate as discussed [here]( https://www.kepware.com/getattachment/ccefc1a5-9b13-41e6-99d9-2b00cc85373e/opc-ua-client-server-easy-guide.pdf).
+f using the [OPC PLC Server Simulator](https://docs.microsoft.com/en-us/samples/azure-samples/iot-edge-opc-plc/azure-iot-sample-opc-ua-server/), be sure to include the “–autoaccept” switch or copy the server.der to the pki.
```
Comment (karok2m) [inline docs/modules/publisher-migrationpath.md]: Seems like the beginning of the sentence has been lost. All that remains is a letter f. Please fix.

Comment (karok2m) [inline docs/modules/publisher-migrationpath.md]: When talking about `pki`, we should mention that command-line argument for configuring it is `PkiRootPath`.

Comment (dacolgit) [reply docs/modules/publisher-migrationpath.md]: fixed

Comment (dacolgit) [reply docs/modules/publisher-migrationpath.md]: added

**Review by karok2m — APPROVED** [review]

**Review by cristipogacean — APPROVED** [review]

**Pull request merged**